src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBroker.cs

```diff
@@ -58,12 +52,10 @@ public async Task<MsalTokenResponse> AcquireTokenUsingBrokerAsync(Dictionary<str
         {
             _androidBrokerTokenResponse = null;
             _correlationId = AndroidBrokerHelper.GetValueFromBrokerPayload(brokerPayload, BrokerParameter.CorrelationId);
-            //Need to disable warning for non awaited async call.
+
             try
             {
-#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
-                new TaskFactory().StartNew(() => AcquireTokenInternalAsync(brokerPayload));
-#pragma warning restore CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
+                await Task.Run(() => AcquireTokenInternalAsync(brokerPayload)).ConfigureAwait(false);
```

Comment:
Exceptions were getting lost when I used TaskFactory().StartNew() so i had to revert
@bgavrilMS

Reply:
Why not just `await AcquireTokenInteranalAsync` ?